flow-typed/npm/lovefield_v2.x.x.js

```diff
@@ -121,12 +121,6 @@ declare module 'lovefield' {
     toSql(): string;
   }
 
-  declare interface Bob {
```

Comment:
Explanation: I had this in for testing some weird flow issues. Accidentally pushed it to github